index.html

```diff
@@ -72,27 +72,27 @@ <h4>Skills</h4>
                 </ul>
             </div>
 
-        </div>
-        <div class="social-icons">
+        </section>
+        <section class="social-icons">
             <ul>
                 <li><i class="fa-brands fa-github"></i></li>
                 <li><i class="fa-brands fa-linkedin-in"></i></li>
                 <li><i class="fa-brands fa-github"></i></li>
                 <li><i class="fa-brands fa-angellist"></i></li>
                 <li><i class="fa-solid fa-m"></i></li>

```

Comment:
- [OPTIONAL] It would be better to wrap those `i` tags into `a `tags to make them linkable